stories/Gauge.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-children-prop */
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, number, text, select, withKnobs } from '@storybook/addon-knobs';
+import { Gauge } from '@textkernel/oneui';
+import { CONTEXTS } from '@textkernel/oneui/constants';
+
+storiesOf('Molecules|Gauge', module)
+    .addDecorator(withKnobs)
+    .add('Gauge', () => {
+        const isProgressLoading = boolean('isProgressLoading', false);
+        const isContentLoading = boolean('isContentLoading', false);
+
+        return (
+            <div style={{ display: 'flex' }}>
+                <div style={{ width: '33%' }}>
+                    <Gauge
+                        context={select('Context', CONTEXTS, CONTEXTS[1])}
+                        percentage={number('Percentage', 33)}
+                        note={isContentLoading ? '—' : 'Low'}
```

Comment:
It took me sometime to test that it will work with tooltips and other complex things. Maybe you could add one such example